modules/lightning_features/lightning_media/lightning_media.info.yml

```diff
@@ -1,10 +1,10 @@
 name: Lightning Media
 type: module
-description: 'Provides install and configuration for great media authoring experiences.'
+description: Slick media handling for Lightning. So cool you'll make the A/C jealous.
 core: 8.x
-package: lightning
+package: Lightning
 dependencies:
-  - embed
   - entity_embed
-  - entity_browser
-  - file_browser
+  - media_entity
+  - rest
+  - serialization
```

Comment:
As discussed, I think all of the media entity plugins need to be requirements here. Please add:
- media_pinkeye
- media_entity_slideshow
- media_entity_instagram
- media_entity_twitter
- media_entity_image